tenzir/functional-test/data/syslog/cef-over-syslog.log

```diff
@@ -0,0 +1,10 @@
+Nov 13 16:59:59 host123 FOO: CEF:0|FORCEPOINT|Firewall|6.6.1|0|Generic|0|deviceExternalId=Master FW node 1 dvc=10.1.1.40 dvchost=10.1.1.40 msg=log server connection established deviceFacility=Logging System rt=Jan 17 2020 08:52:10
```

Comment:
I don't really know which is more prevalent, but at least here: https://help.deepsecurity.trendmicro.com/12_0/aws/Events-Alerts/syslog-parsing.html the `"CEF"` is working as the syslog TAG, so there's no `"FOO: "` before the CEF body.

The `syslog` parser deals with this case correctly, but `read syslog | parse content --into message cef` wouldn't work, because `message` doesn't contain the `CEF:`. This is something to be dealt with elsewhere, nevertheless.